ui/packages/app/src/components/Accordion/Accordion.svelte

```diff
@@ -14,14 +14,39 @@
 
 <div
 	id={elem_id}
-	class="p-3 border border-gray-200 dark:border-gray-700 rounded-lg flex flex-col gap-3 hover:border-gray-300 dark:hover:border-gray-600 transition"
-	class:hidden={!visible}
+	class="container border-gray-200 dark:border-gray-700 hover:border-gray-300 dark:hover:border-gray-600 "
+	style:display={visible ? "block" : "none"}
 >
-	<div on:click={toggle} class="w-full flex justify-between cursor-pointer">
+	<div on:click={toggle} class="label-wrap">
 		<span>{label}</span>
-		<span class:rotate-90={!_open} class="transition">▼</span>
+		<span style:transform={_open ? "rotate(0)" : "rotate(90deg)"} class="icon"
+			>▼</span
+		>
 	</div>
 	<Column visible={_open}>
 		<slot />
 	</Column>
 </div>
+
+<style>
+	.container {
+		display: flex;
+		flex-direction: column;
```

Comment:
There is a weird grey bar under the `count` and `radio` components of `blocks_layout`

### This branch
![image](https://user-images.githubusercontent.com/41651716/209225129-e8178bf4-a164-4cdf-8319-f7443b9968ec.png)

### 3.15.0
![image](https://user-images.githubusercontent.com/41651716/209225169-def65c2f-5a0e-4a4c-81c0-65d729411936.png)